--- vlas.f.orig	2026-09-13 07:16:48
+++ vlas.f	2026-09-13 07:16:48
@@ -947,7 +947,7 @@
          r2=dsqrt(r2/(npart-nbcap-out))
          rc2=dsqrt(rc2/(npart-nbcap))
          open (3,file='moye.dat',status='unknown')
-         write (3,'(I,3e15.7)') nb,r2,rc2,spill
+         write (3,'(I10,3e15.7)') nb,r2,rc2,spill
          close(3)
       else
          do k=1,10
@@ -982,9 +982,9 @@
          rc2=dsqrt(rc2/(npart-nbcap))
          open (3,file='moye.dat',status='old')
          do j=1,nb-1
-            read(3,'(I,3e15.7)')
+            read(3,'(I10,3e15.7)')
          end do
-         write (3,'(I,3e15.7)') nb,r2,rc2,spill
+         write (3,'(I10,3e15.7)') nb,r2,rc2,spill
          close(3)
       end if
       end 
@@ -1572,7 +1572,7 @@
       dumr=0.d0
       do i=0,nx
       end do
-      titrec='/home/tempo3/plagne/'//titre 
+      titrec='out/                '//titre 
       open (2,file=titrec,status='unknown')
       print*,'trec=',titrec
 c      print*,'plot x=0 => entrez 1'
@@ -1676,7 +1676,7 @@
       real*8 rmax,pmax,rhorp(nrmax,npmax),dumx,dumy
       integer nr,np,dumi,i,j
       open (1,file='frp.d',status='unknown')
-      read(1,'(3I)') nr,np,dumi
+      read(1,'(3I10)') nr,np,dumi
       print*,'dumi',dumi
       do i=1,nr
          do j=1,np
@@ -2243,7 +2243,7 @@
       dumi=0
       dumj=0
       dumr=0.d0
-      titrec='/home/tempo3/plagne/'//titre 
+      titrec='out/                '//titre 
       open (2,file=titrec,status='unknown')
       print*,'trec=',titrec
       eps=1.d-5
